q2cli/handlers.py

```diff
@@ -310,12 +310,25 @@ def get_click_options(self):
 
     def get_value(self, arguments, fallback=None):
         import qiime2
-
+        import qiime2.sdk.util
         path = self._locate_value(arguments, fallback)
+
         if path is None:
             return None
+
+        try:
+            # check to see if path is an artifact
+            art = qiime2.Artifact.load(path)
```

Comment:
Ideally we'd have an API to check whether it's a valid QIIME 2 artifact or visualization (I created an issue [here](https://github.com/qiime2/qiime2/issues/259)) but this is fine for now. It'd be handy to have a TODO comment linking to the issue I created.